dp_cluster_reg/config.py

```diff
@@ -0,0 +1,14 @@
+import os
```

Comment:
This file is not pulling its weight. I'd suggest we move the SERVICE_CONF_DIR to some Knox related class. And the Memorized User variable to userinput and eliminate this class.